flax/linen/module.py

```diff
@@ -1496,7 +1519,8 @@ def init(self,
       jit_init(rng, jnp.ones(input_shape, jnp.float32))
 
     Args:
-      rngs: The rngs for the variable collections.
+      rngs: The rngs for the variable collections. This argument will also
+        accept a single rng key as well.
```

Comment:
Nit: since it's an acronym

```suggestion
      rngs: The RNGs for the variable collections. This argument will also
        accept a single RNG key as well.
```

(or "PRNGs")